python/pipeline/preprocess.py

```diff
@@ -130,6 +130,21 @@ def estimate_num_components_per_slice(self):
 
             return int(round(num_components))
 
+        def estimate_neuron_size_in_pixels(self):
+            """ Estimates the size of a neuron in the scan (in pixels). Assumes neurons
+             are 15 x 15 microns."""
```

Comment:
I changed names in the new version to estimate_soma_radius_in_pixels and now it returns a tuple of radius(es?), I left the estimate of the neuron at 14 (thus radius at 7) because if I put less it says neuron are smaller than they actually are in the scan (tested in three different scans)